.github/workflows/tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         run: composer require symfony/phpunit-bridge:^5.4 --dev --prefer-dist --no-interaction --no-progress
 
       - name: Install dependencies
-        uses: nick-invision/retry@v1
```

Comment:
Do we have the same on SF1? Not sure why using an action here 🤔

Reply:
Nope, neither in sf1 nor in doctrine1. Not sure if this is required anymore as well. Think we can drop it.